src/style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  left: 5px;
+  margin-top: 15px;
+  width: 100%;
+  top: 3px;
```

Comment:
You don't need to use `top` and `left` here. You should achieve correct placing by using `margin`.